src/TSID/include/BipedalLocomotion/TSID/TaskSpaceInverseDynamics.h

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file TaskSpaceInverseDynamics.h
+ * @authors Ines Sorrentino
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_TASK_SPACE_INVERSE_DYNAMICS_H
+#define BIPEDAL_LOCOMOTION_TASK_SPACE_INVERSE_DYNAMICS_H
+
+#include <BipedalLocomotion/System/Advanceable.h>
+#include <BipedalLocomotion/System/LinearTask.h>
+
+#include <iDynTree/KinDynComputations.h>
+#include <manif/SE3.h>
+
+namespace BipedalLocomotion
+{
+
+namespace TSID
+{
+
+/**
+ * State of the TaskSpaceInverseDynamics
+ */
+ struct ContactWrench
+{
+    iDynTree::FrameIndex frameIndex; /**< Frame used to express the contact wrench */
+    System::VariablesHandler::VariableDescription variable; /**< Variable describing the contact
+                                                                wrench */
+};
+struct TSIDState
+{
+    manif::SE3d::Tangent baseAcceleration; /**< Mixed spatial acceleration of the base */
```

Comment:
```suggestion
    manif::SE3d::Tangent baseAcceleration; /**< Mixed acceleration of the base */
```

`spatial` is usually right trivialized.